modules/hub/scripts/kubectl_wrapper.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
I dont think we are using this script for hub registration?

Reply:
Removed